deepfence_frontend/apps/dashboard/src/features/settings/pages/EmailConfiguration.tsx

```diff
@@ -1,4 +1,4 @@
-import { Suspense, useEffect, useState } from 'react';
+import { Suspense, useState } from 'react';
```

Comment:
delete email configuration -> add it again -> try deleting again, only success message shows instead of confirm modal

Reply:
fixed the issue